src/kernels/composable_kernel/include/kernel_algorithm/gridwise_group_convolution_forward_implicit_gemm_v4r4_xdlops_gnchw_gkcyx_gnkhw.hpp

```diff
@@ -0,0 +1,182 @@
+#ifndef CK_GRIDWISE_GROUP_CONVOLUTION_FORWARD_IMPLICIT_GEMM_V4R4_XDLOPS_GNCHW_GKCYX_GNKHW_HPP
```

Comment:
Please remove "group" from filename as gnchw already indicates group functionality. Secondly, we want to condense the names of igemm files so that it is easy to identify files in visual editor/cscope. Right now, due to big names and redundant info, it is getting harder.